src/EventListener/ShippingExportEventListener.php

```diff
@@ -82,26 +83,26 @@ public function exportShipment(ResourceControllerEvent $event): void
 
             $this->saveShippingLabel($shippingExport, $labelContent, 'pdf');
         } catch (SoapFault $exception) {
-            $this->flashBag->add(
+            $this->requestStack->getSession()->getBag('flashes')->add(
```

Comment:
I would assign the flashbag to a variable

Reply:
Fixed